config/nginx.conf.erb

```diff
@@ -2,6 +2,10 @@ daemon off;
 #Heroku dynos have at least 4 cores.
 worker_processes <%= ENV['NGINX_WORKERS'] || 4 %>;
 
+set_real_ip_from 127.0.0.1;
```

Comment:
Since Heroku can guarantee that all connections to us are coming from their router (and it's not possible to list out all their IP addresses), I think this should be `0.0.0.0/0`.

Reply:
We will have to find a better solution when we put a CDN in front of the API, but I agree that this is fine in the short term.

Reply:
Whoops, you're right. You can find a little bit of documentation on it on the docs for internally routed apps, but mostly it's just experience and the fact that it wouldn't make sense for them to use any IP outside of the range laid out in RFC 1918